model/src/main/java/org/cbioportal/genome_nexus/model/SignalAnnotation.java

```diff
@@ -0,0 +1,37 @@
+package org.cbioportal.genome_nexus.model;
+
+
+import java.util.Collections;
+import java.util.List;
+
+public class SignalAnnotation
+{
+    private String license;
+
+    private List<SignalMutation> annotation;
```

Comment:
`SignalMutation` has a lot of data, not sure if we need all of it. We may want to return a partial response here instead.

Reply:
i see, yeah maybe only the fields that we are showing in signaldb.org? 

E.g. penetrance, %, % biallelic, % somatic driver, % germline pathogenic and broken down by cancer type

Reply:
im not sure how much more data there is in signalmutation, so if it's not too crazy, maybe ok. Could you add some example response to the description of the PR + maybe in a test?